sync/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:3.7
+
+WORKDIR /app
+# Only copy specific values instead of the entire directory.
+# We're primarily trying to avoid pulling in config/, since it's confusing that
+# it will not be used.
+COPY sync.py sync.py
+COPY requirements.txt requirements.txt
+
+RUN pip install -r requirements.txt
+
+ENTRYPOINT [ "python3", "sync.py" ]
```

Comment:
you can just use `python` here as you're pulling the python3.7 image...

```
$ docker run -it python:3.7 bash
root@6989d0bbf8d2:/# which python
/usr/local/bin/python
root@6989d0bbf8d2:/# python --version
Python 3.7.7
```

Reply:
While this is also correct, I'd prefer to keep this as python3.

I find that I use Dockerfiles as a guide for running applications locally for development and testing, and we can't assume that the default system python is necessarily python3 (particularly on OSX).

On the image `python` is just a symlink to `python3`, so using either is the same.

```
$ docker run -it python:3.7 bash
root@58befeb9dc6c:/# ls -al $(which python)
lrwxrwxrwx 1 root root 7 May 16 07:18 /usr/local/bin/python -> python3
```